src/anurag/Array.java

```diff
@@ -0,0 +1,98 @@
+package anurag;
+
+public class Array {
+
+	int ar[] = { 10, 5, 30, 20, 90 };
+
+	int displayMin() {
+
+		int min = ar[0];
+
+		for (int index = 0; index < 5; index++) {
```

Comment:
Video sent on skype for this code, pls go through it

Also Check this code for reference
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/318/files